main.rb

```diff
@@ -0,0 +1,51 @@
+require_relative './app'
+
+def main
+  app = App.new
+  loop do
+    display_menu
+    choose_action(app)
+  end
+end
+
+def display_menu
+  puts "\nWelcome to School Library App!"
+  puts "\nPlease choose an option by entering a number:"
+  options = [
+    '1 - List all books',
+    '2 - List all people',
+    '3 - Create a person',
+    '4 - Create a book',
```

Comment:
- [ ] As requested by the previous code reviewer, which is a requirement for this project, kindly fix the `create_book` method. Checkout the screen shot below and kindly fix the issue. 

![create_book_error](https://user-images.githubusercontent.com/65852621/212498419-e373249f-c919-4109-ab7d-dfe96d5c07a4.PNG)

Reply:
@Mihndim2020 , Thank you for your time really!
I am not sure what the issue is because it's working fine from here :point_down: :

![image](https://user-images.githubusercontent.com/99715640/212498692-e2e05a13-3784-4710-bdea-6d34b800d762.png)

Reply:
Oh, I am so sorry. I didn't push after committing my solution